helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/StringRepresentation.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+/**
+ * represents the possible ways avro code generation may represent String fields
+ * in generated code
+ */
+public enum StringRepresentation {
+    CharSequence, String, Utf8
+}
```

Comment:
What I'm saying is, your enum should look like:

```
public enum StringRepresentation {
  CharSequence(() -> {
    Class<?> fieldTypeEnum = Class.forName("org.apache.avro.generic.GenericData$StringType");
    Field charSequenceField = fieldTypeEnum.getDeclaredField("CharSequence");            
    return charSequenceField.get(null);
  }
  ),
  String(() -> {...}),
  Utf8(() -> {...});

  Object datThing;
  StringRepresentation(Object datThing) {
    this.datThing = datThing;
  }
}
```

Reply:
the enum class does not exist in avro 1.4/5, so this would throw a runttime exception running under avro 1.4/5